lib/generators/logidze/install/functions/logidze_logger.sql

```diff
@@ -89,11 +89,25 @@ CREATE OR REPLACE FUNCTION logidze_logger() RETURNS TRIGGER AS $body$
       changes := '{}';
 
       IF (coalesce(current_setting('logidze.full_snapshot', true), '') = 'on') THEN
-        changes = hstore_to_jsonb_loose(hstore(NEW.*));
+        BEGIN
+          changes = hstore_to_jsonb_loose(hstore(NEW.*));
+        EXCEPTION
+          WHEN NUMERIC_VALUE_OUT_OF_RANGE THEN
+            changes = row_to_json(NEW.*)::jsonb;
```

Comment:
Don't we need to stringify object values? Like we do here:https://github.com/palkan/logidze/blob/3521c5f26b0803cd06c541f0fd6524e2380bff27/lib/generators/logidze/install/functions/logidze_snapshot.sql#L19

Otherwise that wouldn't be compatible with hstore version.